Update windows.md

Ompanime opened the pull request:
- Switched "\" to "/" under "Build go-zond (the Execution Engine) with Golang v1.21 or later" to work in MinGW (Bash Terminal)

- Changed "copy" to "cp" under "Build go-zond (the Execution Engine) with Golang v1.21 or later" to work in the MinGW (Bash Terminal)

jplomas commented:
[incidentally the _Git for Windows_ 'Git Bash' terminal is brilliant and well worth a look too!]

Ompanime commented:
Liked the suggestion to make instructions for the MinGW console, but I'm not experienced enough to do that. Instead I corrected the command syntax back to the original, with the exception of building qrysm, where I removed the "=". Replaced MinGW with TDM-GCC to save system resources, and clarified that commands be entered via the Windows Command Line for inexperienced users.
